packages/rockets-next-web/src/app/showroom/layout.tsx

```diff
@@ -3,93 +3,18 @@
 import { ReactNode } from "react";
 import { usePathname, useRouter } from "next/navigation";
 import { ContainerWithDrawer } from "@concepta/react-material-ui";
-import PersonOutlineOutlinedIcon from "@mui/icons-material/PersonOutlineOutlined";
-import BusinessOutlinedIcon from "@mui/icons-material/BusinessOutlined";
 import GroupsOutlinedIcon from "@mui/icons-material/GroupsOutlined";
-import PaymentsOutlinedIcon from "@mui/icons-material/PaymentsOutlined";
-import ExtensionOutlinedIcon from "@mui/icons-material/ExtensionOutlined";
-import HomeOutlinedIcon from "@mui/icons-material/HomeOutlined";
-import ColorLensOutlinedIcon from "@mui/icons-material/ColorLensOutlined";
-import GridOnIcon from "@mui/icons-material/GridOn";
-import CodeIcon from "@mui/icons-material/Code";
 
 export default function Showroom({ children }: { children: ReactNode }) {
   const pathname = usePathname();
   const router = useRouter();
 
   const drawerMenuItems = [
     {
-      id: "/showroom/profile",
-      icon: <PersonOutlineOutlinedIcon />,
-      text: "Profile",
-      onClick: () => router.push("/showroom/profile"),
-    },
-    {
-      id: "/showroom/account",
-      icon: <BusinessOutlinedIcon />,
-      text: "Account",
-      onClick: () => alert("foo"),
-      active: true,
-    },
-    {
-      id: "/showroom/billing",
-      icon: <PaymentsOutlinedIcon />,
-      text: "Billing",
-      onClick: () => alert("foo"),
-    },
-    {
-      id: "/showroom/integrations",
-      icon: <ExtensionOutlinedIcon />,
-      text: "Integrations",
-      onClick: () => alert("foo"),
-    },
-    {
-      id: "/showroom/home",
-      icon: <HomeOutlinedIcon />,
-      text: "Home",
-      onClick: () => router.push("/showroom/home"),
-    },
-    {
-      id: "/showroom/table",
-      icon: <GridOnIcon />,
-      text: "Table",
-      onClick: () => router.push("/showroom/table"),
-    },
-    {
-      id: "/showroom/teamMembers",
+      id: "/showroom/users",
```

Comment:
```suggestion
      id: "/users",
```

Reply:
If I change just the `id`, the Users menu item won't be highlighted when the page is being displayed. Should I move the `users` folder out of the `showroom` directory?

Reply:
Yes

Reply:
we should not use showroom route

Reply:
Got it. In ecn-portal we have the (auth) and (main) route groups and I believe we should follow the same structure here

Reply:
Done!